dashboard/app/views/levels/_admin.html.haml

```diff
@@ -117,15 +119,15 @@
       .admin-report-abuse{ style: 'display: none;' }
         = link_to 'Report Abuse', '/report_abuse'
 
-      #disable-auto-moderation
+      #disable-auto-moderation{'style' => ('display: none;' unless !content_moderation_disabled)}
```

Comment:
Nit: `if content_moderation_disabled` instead of `unless !content_moderation_disabled`